### ISSUE TITLE
Hasura RESTified endpoint supported types (Array)

### ISSUE THREAD

**Issue by makarovivan** [hasura/graphql-engine#6911]
Hi,

I have query with array parameters and it works. I try to create REST endpoint from it.

example query:
```
query getSampleData($start: [Float]!, $finish: [Float]!, $specialType: [SpecialType]!) {
  getSampleData(start: $start, finish: $finish, mobilityType: $mobilityType) {
    result
}
```
it works from GraphiQL

But does not work in Rest:
`curl -X GET https://graphql.graphql.local:443/api/rest/testRoute --data '{ "finish": [30.0, 99.0], "start": [30.0, 99.9], "specialType": ["TYPE1", "TYPE2"] }'`

`{"path":"$","error":"The variable type for the expected variable start was not supported.","code":"bad-request"}`

It seems, REST endpoints does not support Arrays? Or there is something else?

**Comment by jmsegrev** [comment]
I'm having the same problem, had to go back to creating a normal API endpoint that would call hasura with graphql to answer to this type of requests.

**Comment by nagautier** [comment]
Same issue here, I love the new restified stuff, but without array support, it's a bit limited. Would be a big plus to have this :) !

**Comment by BenoitRanque** [comment]
Closing as this bug was fixed and released in 2.1
